src/ch11-Greedy/index.js

```diff
@@ -0,0 +1,42 @@
+const solution_01 = require("./01/solution");
+const solution_02 = require("./02/solution");
+const solution_03 = require("./03/solution");
+const solution_04 = require("./04/solution");
+const solution_06 = require("./06/solution");
+
+const solutions = {
+  getInputs: function (dir) {
```

Comment:
요것도 이렇게 해도 괜찮지 않을까요?

```suggestion
  getInputs(dir) {
```

Reply:
넵 감사합니다. 반영했습니다!